crates/evm/evm/src/executors/fuzz/mod.rs

```diff
@@ -113,6 +116,7 @@ impl FuzzedExecutor {
                         data.traces.push(call_traces);
                         data.breakpoints.replace(case.breakpoints);
                     }
+                    data.logs.extend(case.logs);
```

Comment:
maybe extend only if `show_logs` is true? Then 
```Solidity
        // decide whether to use trace logs or only error logs
        let actual_logs = if show_logs { fuzz_result.logs } else { call.logs };
```

could go away and simply use `fuzz_result.logs`

Reply:
done. only when it's true that needs to be extended.